lighthouse-core/lib/stack-packs.js

```diff
@@ -7,27 +7,39 @@
 
 const stackPacks = require('@lighthouse/stack-packs');
 
+const stackPacksToInclude = [{
```

Comment:
`stackPackSafelist`

Reply:
> `stackPackSafelist`

"safe list" seems confusing to me in this context, `stackPacksToInclude` seems more like what we mean to me